x-pack/plugins/infra/public/components/asset_details/asset_details.stories.tsx

```diff
@@ -9,13 +9,22 @@ import React, { useState } from 'react';
 import { EuiButton } from '@elastic/eui';
 import type { Meta, Story } from '@storybook/react/types-6-0';
 import { i18n } from '@kbn/i18n';
+import type { DataViewField } from '@kbn/data-views-plugin/common';
+import type { DataView } from '@kbn/data-views-plugin/public';
```

Comment:
Could we import both types from `/public`?

Reply:
Oh yeah, `DataViewField` is also available in `public`, I guess the import helper got it from `common`. Fixed 👍